platform/engineering/design-docs/2020-08-12-mobile-ssoe-auth-design-doc.md

```diff
@@ -0,0 +1,266 @@
+# Mobile SSOe Auth Design Doc
+
+**Author(s):** Alastair Dawson, Jonathan Julian  
+**Last Updated:** August 12th, 2020  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** Andrew Gunsch, John Paul Ashenfelter, Michael Fleet 
+
+## Overview
+
+### Objective
+Allow VA Mobile App users, authenticated via the IAM SSOe identity provider, to consume endpoints in api.va.gov
+
+### Background
+A flagship mobile app for the VA is being developed to augment services available on va.gov with unique mobile features 
+such as notifications and biometric sign-in. The VA’s Identity and Access Management System (IAM) maintains a Single Sign On 
+identity provider/service for external apps (SSOe). Web clients can currently establish VA.gov sessions from SAML, Okta, 
+and across vites via SSO. To enable the mobile app to consume api.va.gov endpoints we’ll allow mobile SSOe signed requests 
+to also create and use VA.gov sessions.
+
+Currently in vets-api all web client session (from va.gov) logic is managed in the `AuthenticationAndSSOConcerns` concern 
+that is mixed in to the main `ApplicationController`.  API sessions (from lighthouse) are managed in the `OpenidApplicationController`.
+
+The main difference between the two are that session extension/expiration for web requests are handled in the app while API 
+session TTL is part of the token payload.
+
+### High Level Design
+When launching the mobile app for the first time a call is made to IAM OAuth service's `../authorise` endpoint. 
```

Comment:
what data does it send to `../authorise`?

Does "When launching the mobile app for the first time" imply that user credentials aren't needed for this step?